docs/onboard.md

```diff
@@ -0,0 +1,86 @@
+# Onboard
+
+## Vendor
```

Comment:
Do we have a page that describes the differences of Public Officer accounts and Vendor accounts?

Reply:
those are content damien and denise have plans to extract the content from private docs.

this set of changes by komal is purely on onboarding process